### DISCUSSION TITLE
How to change time format in cockpit?

### DISCUSSION THREAD

**Discussion by EvelynOctavia** [cockpit-project/cockpit#20715] [Q&A] [answered]
In cockpit, the time shown in "last login" and "system time" is in 12 hour format. How do I change it to 24 hour format? Thanks

**Comment by martinpitt** [comment, marked as answer]
You can't -- this is formatted by the standard browser [Intl.DateTieFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) API. I suppose you could change your prefered language to en-GB?
